test/dlc_tests/container_tests/bin/benchmark/run_pytorch_training_performance_gpu_inductor

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[0])' | tr -d "'")
+if [ "$PYTHON_VERSION" -eq 2 ]
+then
+  exit 0
+fi
+HOME_DIR=/test/benchmark
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs
+
+mkdir -p ${HOME_DIR}
+mkdir -p ${BIN_DIR}
+mkdir -p ${LOG_DIR}
+
+python -c "import sys;
+assert sys.version_info < (3, 7);"
+PYTHON_VERSION_CHECK=`echo $?`
+
+set -e
+cd $BIN_DIR
+rm -rf pytorch
+# tmp fix for numpy deprecations, PR raised: https://github.com/HewlettPackard/dlcookbook-dlbs/pull/19
+git clone https://github.com/pytorch/pytorch.git pytorch
+
+cd pytorch
+
+pip install -U numpy
+pip install deepspeed==0.8.2
```

Comment:
Remove deepspeed if unused. It's presence can cause other packages to behave differently. Eg: transformers